src/metabase/driver/sql_jdbc/execute.clj

```diff
@@ -571,14 +571,14 @@
       (execute-prepared-statement! driver st))))
 
 (defmethod read-column-thunk :default
-  [driver ^ResultSet rs rsmeta ^long i]
+  [driver ^ResultSet rs rsmeta ^Long i]
   (let [driver-default-method (get-method read-column-thunk driver)]
     (if-not (= driver-default-method (get-method read-column-thunk :default))
       ^{:name (format "(read-column-thunk %s)" driver)} (driver-default-method driver rs rsmeta i)
       ^{:name (format "(.getObject rs %d)" i)} (fn []
                                                  (.getObject rs i)))))
 
-(defn- get-object-of-class-thunk [^ResultSet rs, ^long i, ^Class klass]
+(defn- get-object-of-class-thunk [^ResultSet rs ^Long i ^Class klass]
```

Comment:
Also unrelated but `^long` is wrong here since I'm 99% sure we're passing around boxed `Long`s rather than raw 64-bit integers